shell/platform/windows/window_unittests.cc

```diff
@@ -63,9 +69,11 @@ TEST(MockWindow, OnImeCompositionCompose) {
 }
 
 TEST(MockWindow, OnImeCompositionResult) {
-  MockTextInputManager* text_input_manager = new MockTextInputManager();
+  auto windows_proc_table = std::make_unique<MockWindowsProcTable>();
+  auto* text_input_manager = new MockTextInputManager();
   std::unique_ptr<TextInputManager> text_input_manager_ptr(text_input_manager);
```

Comment:
Can't remember why this is using new and not make_unique() but definitely feel free to clean it up.

Reply:
Sounds good. I'll do it in a follow-up pull request. I need to merge this one back into the beta release branch, which is already has hairy merge conflicts 😅